src/YoloDev.Expecto.TestSdk/YoloDev.Expecto.TestSdk.fsproj

```diff
@@ -16,7 +15,6 @@
     <Compile Include="discovery.fs" />
     <Compile Include="execution.fs" />
     <Compile Include="adapter.fs" />
-    <Compile Include="main.fs" />
```

Comment:
Same as above. Is this correct for all targets?

Reply:
That's what I believe - the arguments are the same as in the previous comment.